### PR TITLE
Connect UI

### PR DESCRIPTION
**Pull request by imaximumGit**
All the view for the connection are ready and a basic navigation allow to navigate between the components.
For now the app is in Expo-CLI so the it's unable to use native components (Camera). The QR code reader will be implement later.

### CODE REVIEW

File: PopApp/App.js [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+import Navigation from './Navigation/Navigation'
+import { NavigationContainer } from '@react-navigation/native';
+
+/*
+* The start point of the app
+*
+* It open the navigation component in a safeAreaProvider to be able use
+* SafeAreaView in order to resolve issue with status bar
+*
+*  The Platform.OS is to put the statusBar in IOS in black, otherwise it is not readable
```
Comment (lhmerino) [inline PopApp/App.js:15]: Great comment!

**Review by lhmerino — APPROVED** [review]
Great job! Wonderful comments; I like that you said what it does now and what it should do in the future as it provides direction.

**Comment by lhmerino** [comment]
Looks like you will need to resolve conflicts before you can merge.

**Review by RaulinN — APPROVED** [review]

**Review by RaulinN — COMMENTED** [review]
Added a React import in ConnectUnapprove for the project to compile properly on iOS at least

**Pull request merged**